src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx

```diff
@@ -103,7 +103,7 @@ const PanelContainer = styled('div')`
   display: flex;
   flex-direction: column;
   z-index: ${p => p.theme.zIndex.sidebarPanel};
-  background: ${p => p.theme.white};
+  background: ${p => p.theme.backgroundSecondary};
```

Comment:
backgroundSecondary is gray. Could you change this to background instead? It's white to match onboarding.

Reply:
Thank you for the feedback. While I agree that white cards on a white background is more "modern" (Material Design etc), I feel that white cards on a gray background is more consistent with the rest of the product, such as Projects, Releases, and Discover. However, if you feel strongly about it, I can change it to `background` instead.

Reply:
I do feel strongly about this. This is a continuation of the existing onboarding flow UI and I've been making these changes for awhile now.